srcpkgs/xcb-util-cursor/template

```diff
@@ -1,17 +1,17 @@
-# Template build for 'xcb-util-cursor'.
+# Template file for 'xcb-util-cursor'
 pkgname=xcb-util-cursor
-version=0.1.3
+version=0.1.4
 revision=1
 build_style=gnu-configure
 configure_args="--disable-static"
 hostmakedepends="pkg-config gperf m4"
 makedepends="xcb-util-renderutil-devel xcb-util-image-devel"
 short_desc="XCB cursor library (libxcursor port)"
 maintainer="Orphaned <orphan@voidlinux.org>"
-homepage="http://xcb.freedesktop.org"
 license="MIT"
-distfiles="http://xcb.freedesktop.org/dist/${pkgname}-${version}.tar.bz2"
-checksum=05a10a0706a1a789a078be297b5fb663f66a71fb7f7f1b99658264c35926394f
+homepage="http://xcb.freedesktop.org"
+distfiles="http://xcb.freedesktop.org/dist/xcb-util-cursor-${version}.tar.xz"
```

Comment:
```suggestion
homepage="https://xcb.freedesktop.org"
distfiles="https://xcb.freedesktop.org/dist/xcb-util-cursor-${version}.tar.xz"
```

Reply:
Done, thanks for the feedback. Sorry for the wait.